protos/protos/public/common/CommonService.proto

```diff
@@ -69,6 +69,7 @@ message ModelDBResourceEnum {
         DATASET_VERSION = 6;
         DASHBOARD = 7;
         REPOSITORY = 8;
+        REGISTERED_MODEL = 9;
```

Comment:
Why is this in common?

Reply:
i can not figure out a good reason for it right now, but we/you did move it here right during the initial private //public restructure . https://github.com/VertaAI/modeldb/pull/736

Reply:
Something to look at later. Not a blocker.